packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -27,98 +18,146 @@ const address: Address = {
   phone: '514 xxx xxxx',
 };
 
+afterEach(fetch.restore);
+
 describe('updateLocale()', () => {
   beforeEach(() => {
-    fetch.mock(
-      'https://country-service.shopifycloud.com/countries/CA?locale=fr',
-      {data: CanadaFr},
-    );
-
-    fetch.mock(
-      'https://country-service.shopifycloud.com/countries/CA?locale=en',
-      {data: CanadaEn},
-    );
+    fetch.mock((url, options) => {
+      if (
+        url === GRAPHQL_ENDPOINT &&
+        options.method === 'POST' &&
+        JSON.parse(options.body).operationName === 'country' &&
+        JSON.parse(options.body).variables.locale === 'ja'
+      ) {
+        return true;
+      }
+      return false;
+    }, countryJP_JA);
```

Comment:
Because this is long and not super readable, can we refactor that with something like:
```ts
mockApiCall('country', 'ja', countryJP_JA)
```